app/config/config_prod.yml

```diff
@@ -37,3 +39,13 @@ monolog:
         newrelic:
             type: newrelic
             level: error
+
+nelmio_security:
+    csp:
+        enforce:
+            default-src:
+              - 'https:'
+    forced_ssl:
+        hsts_max_age: 31536000 # 1 year
+        hsts_preload: true
+        hsts_subdomains: true
```

Comment:
We'll keep this in the application for now, but remove the `hsts_subdomains` and `hsts_preload` option (see ELPP-2948).